http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -537,16 +537,25 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig,
             .createClusterIamRoles(existingCluster.googleProject, existingCluster.serviceAccountInfo)
             .unsafeToFuture()
 
+          _ <- clusterHelper
+            .updateDataprocImageGroupMembership(existingCluster.googleProject, createCluster = true)
+            .unsafeToFuture()
+
           // Resize the cluster
           _ <- gdDAO.resizeCluster(existingCluster.googleProject,
                                    existingCluster.clusterName,
                                    updatedNumWorkersAndPreemptibles.left,
                                    updatedNumWorkersAndPreemptibles.right) recoverWith {
             case gjre: GoogleJsonResponseException =>
-              // Typically we will revoke this role in the monitor after everything is complete, but if Google fails to resize the cluster we need to revoke it manually here
+              // Typically we will revoke this role in the monitor after everything is complete, but if Google fails to
+              // resize the cluster we need to revoke it manually here
               clusterHelper
                 .removeClusterIamRoles(existingCluster.googleProject, existingCluster.serviceAccountInfo)
                 .unsafeToFuture()
+              // Remove member from the Google Group that has the IAM role to pull the Dataproc image
+              clusterHelper
```

Comment:
This future should probably be chained with the one above it. Maybe something like:
```
val cleanupIO = for {
  _ <- clusterHelper.removeClusterIamRoles(existingCluster.googleProject, existingCluster.serviceAccountInfo)
  _ <- clusterHelper.updateDataprocImageGroupMembership(existingCluster.googleProject, createCluster = false)
  _ <- IO(logger.error(s"Could not successfully update cluster ${existingCluster.projectNameString}", gjre))
  err <- IO.raiseError(InvalidDataprocMachineConfigException(gjre.getMessage))
} yield err
cleanupIO.unsafeToFuture
```

Reply:
minor, `err` is not needed.
```
val cleanupIO = for {
  _ <- clusterHelper.removeClusterIamRoles(existingCluster.googleProject, existingCluster.serviceAccountInfo)
  _ <- clusterHelper.updateDataprocImageGroupMembership(existingCluster.googleProject, createCluster = false)
  _ <- IO(logger.error(s"Could not successfully update cluster ${existingCluster.projectNameString}", gjre))
  _ <- IO.raiseError(InvalidDataprocMachineConfigException(gjre.getMessage))
} yield ()
```

Reply:
👍